pom.xml

```diff
@@ -203,28 +203,51 @@
             <target>1.8</target>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-javadoc-plugin</artifactId>
+          <version>2.10.4</version>
+          <inherited>false</inherited>
+          <configuration>
+            <reportOutputDirectory>docs</reportOutputDirectory>
+            <destDir>apidocs</destDir>
+          </configuration>
+          <executions>
+            <execution>
+              <id>aggregate</id>
+              <phase>site</phase>
+              <goals>
+                <goal>aggregate</goal>
```

Comment:
This was setup for just the parent project so that running javadoc:aggregate would generate combined docs for all projects. I don't think moving it to pluginManagement is the right thing to do

Reply:
Personally I like the configuration of a plugin do be done in the plugin management part and only include the group and artifact id in the build section. It looks neat to me.

Reply:
It gets applied to the children projects then though, right? And we want the parent and children to have different configuration here. I'm not much of a Maven expert...